QUDAConfig.cmake.in

```diff
@@ -7,6 +7,10 @@ set(QUDA_MPI @QUDA_MPI@)
 set(QUDA_QIO @QUDA_QIO@)
 set(QUDA_OPENMP @QUDA_OPENMP@)
 set(QUDA_QDPJIT @QUDA_QDPJIT@)
+set(QUDA_VERSION @CMAKE_PROJECT_VERSION@)
```

Comment:
This seems to introduce a new `QUDA_VERSION` variable which we already have defined in the source code.  What am I missing here?

Reply:
Ok I see, I can get it from `quda_constants.h`. I missed that. Then it is not needed to be exposed here. I will remove it